Blaise-Shyaka/AnnounceIT
# [#171205115] Integrate database for signin API endpoint

# What does this PR do?

It integrates postgres database into the signin API endpoint

# Description of tasks to be completed

- Write tests for the database integration
- Write queries for the /auth/signin endpoint

# How should this be manually tested?

- Clone this repository and in the root directory run "npm run test" command
- 33 tests should pass

# What are the related pivotal tracker stories?

#171205115
